vllm/compilation/decorators.py

```diff
@@ -302,7 +348,16 @@ def patched_inline_call(parent, func, args, kwargs):
             with patch.object(InliningInstructionTranslator, 'inline_call',
                               patched_inline_call), torch._dynamo.config.patch(
                                   **dynamo_config_patches):
-                output = self.compiled_callable(*args, **kwargs)
+                if envs.VLLM_USE_AOT_COMPILE:
+                    self.aot_compiled_fn = self.aot_compile(*args, **kwargs)
+                    output = self.aot_compiled_fn(self, *args, **kwargs)
+                    assert aot_compilation_path is not None
+                    assert cache_dir is not None
+                    os.makedirs(cache_dir, exist_ok=True)
+                    self.aot_compiled_fn.save_compiled_function(
+                        aot_compilation_path)
+                else:
+                    output = self.compiled_callable(*args, **kwargs)
```

Comment:
Can you explain if/how VLLM_USE_AOT_COMPILE=1 changes what is stored in ~/.cache/vllm/torch_compile_cache ?